pkg/csi/cinder/nodeserver.go

```diff
@@ -426,6 +426,25 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 	return &csi.NodeStageVolumeResponse{}, nil
 }
 
+// formatAndMountRetry attempts to format and mount a device at the given path.
+// If the initial mount fails, it rescans the device and retries the mount operation.
+func (ns *nodeServer) formatAndMountRetry(devicePath, stagingTarget, fsType string, options []string) error {
+	m := ns.Mount
+	err := m.Mounter().FormatAndMount(devicePath, stagingTarget, fsType, options)
+	if err != nil {
+		klog.Errorf("Initial format and mount failed: %v. Attempting rescan.", err)
```

Comment:
if we accept this rescan logic, this can only be Infof log as it's normal case now

Reply:
Thanks for pointing this out! I will also change the rescan error to a info level log as there are devices that do not have the interface so no need to log an error in those cases either